api/src/main/java/au/org/aodn/nrmn/restapi/controller/mapping/ObservableItemGetDtoMapperConfig.java

```diff
@@ -0,0 +1,43 @@
+package au.org.aodn.nrmn.restapi.controller.mapping;
+
+import au.org.aodn.nrmn.restapi.dto.observableitem.ObservableItemGetDto;
+import au.org.aodn.nrmn.restapi.model.db.AphiaRelType;
+import au.org.aodn.nrmn.restapi.model.db.LengthWeight;
+import au.org.aodn.nrmn.restapi.model.db.ObservableItem;
+import au.org.aodn.nrmn.restapi.model.db.ObsItemType;
+import au.org.aodn.nrmn.restapi.repository.ObsItemTypeRepository;
+import au.org.aodn.nrmn.restapi.repository.ObservableItemRepository;
+
+import org.modelmapper.Converter;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ObservableItemGetDtoMapperConfig {
+
+    @Autowired
+    ObservableItemGetDtoMapperConfig(ObsItemTypeRepository obsItemTypeRepository, ObservableItemRepository observableItemRepository, ModelMapper modelMapper) { 
+        
+        Converter<ObsItemType, String> toObsItemTypeName = ctx -> ctx.getSource() == null ? null : ctx.getSource().getObsItemTypeName();
+        Converter<ObsItemType, Integer> toObsItemTypeId = ctx -> ctx.getSource() == null ? null : ctx.getSource().getObsItemTypeId();
+        Converter<AphiaRelType, String> toAphiaRelTypeName = ctx -> ctx.getSource() == null ? null : ctx.getSource().getAphiaRelTypeName();
+        Converter<Integer, String> toSupersededByIDs = ctx -> observableItemRepository.findOneProjectedById(ctx.getSource()).getSupersededIDs();
+        Converter<Integer, String> toSupersededByNames = ctx -> observableItemRepository.findOneProjectedById(ctx.getSource()).getSupersededNames();
```

Comment:
I'd use a postConverter so that we only perform the lookup once.   See the species mapper.